schema_editor/test/spec/views/geography/list-directive.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+describe('ase.views.geography: geoList', function () {
+
+    beforeEach(module('ase.mock.resources'));
+    beforeEach(module('ase.templates'));
+    beforeEach(module('ase.views.geography'));
+
+    var $compile;
+    var $httpBackend;
+    var $rootScope;
+    var RecordTypes;
```

Comment:
In all of these tests, you can get rid of RecordTypes. 
Also, you should probably add a test for the geography service.